test/graphgym/example_node/0/test/stats.json

```diff
@@ -0,0 +1,4 @@
+{"epoch": 0, "loss": 1.8685, "lr": 0, "params": 23880, "time_iter": 0.0194, "accuracy": 0.361}

```

Comment:
Thank you for confirming. It looks like you accidentally pushed these logs here.

Reply:
I removed these logs